Source/RapyutaSimulationPlugins/Public/Tools/SimulationState.h

```diff
@@ -15,6 +15,18 @@
 
 #include "SimulationState.generated.h"
 
+USTRUCT()
+struct RAPYUTASIMULATIONPLUGINS_API FActorsWithTag
+{
+    GENERATED_BODY()
+
+    UPROPERTY()
+    FName tag;

```

Comment:
is this tag variable necessary if FActorsWithTag is used in a TMap?
Or are there future case where it won't be the case?

Reply:
I think, I can remove